packages/blade/src/components/Input/SelectInput/SelectInput.tsx

```diff
@@ -212,7 +205,10 @@ const _SelectInput = (
       {!isReactNative() ? (
         <VisuallyHidden>
           <input
-            ref={inputRef as React.Ref<HTMLInputElement>}
+            onFocus={() => {
```

Comment:
why do we do this onFocus? isn't it handled already?

Reply:
Check the old diff, it's as same as the previous logic just that we were using the ref's focus event to trigger this.